redhat-developer/developers.redhat.com
# [DEVELOPER-5756] Add Topics and Tags to all content types.

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5756

### Verification Process
1. Edit all content types
2. Go to Fields tab and look in "Purpose Attributes" field group
3. See "Topics" is first field
4. See "Control Tags" is last field

### Screenshot
![Screen Shot 2019-05-08 at 7 27 27 AM](https://user-images.githubusercontent.com/41014351/57400267-52ce6c80-7190-11e9-8190-4fba4f8030bb.png)
![Screen Shot 2019-05-08 at 7 27 37 AM](https://user-images.githubusercontent.com/41014351/57400268-52ce6c80-7190-11e9-9bdf-48195bd054e9.png)
